src/Microsoft.PowerShell.Commands.Utility/resources/Debugger.resx

```diff
@@ -142,7 +142,7 @@
     <value>You cannot debug the default host Runspace using this cmdlet. To debug the default Runspace use the normal debugging commands from the host.</value>
   </data>
   <data name="RunspaceDebuggingNoHostRunspaceOrDebugger" xml:space="preserve">
-    <value>Cannot debug Runspace. The host has no debugger. Try debugging the Runspace inside the Windows PowerShell console or the Windows PowerShell ISE, both of which have built-in debuggers.</value>
+    <value>Cannot debug Runspace. The host has no debugger. Try debugging the Runspace inside the PowerShell Core console or the Visual Studio Code, both of which have built-in debuggers.</value>
```

Comment:
>or the [](start = 122, length = 6)

Suggestion: 'or the Visual Studio Code' is rather cumbersome to read IMHO.
How about 'or with Visual Studio Code.'

Reply:
Fixed.